agent/php_execute.c

```diff
@@ -1654,10 +1904,36 @@ void nr_php_observer_fcall_begin(zend_execute_data* execute_data) {
    * nr_php_execute
    * nr_php_execute_show
    */
+  zval* func_return_value = NULL;
+  if (nrunlikely(NULL == execute_data)) {
+    return;
+  }
+
+  NRPRG(php_cur_stack_depth) += 1;
 
-  if (NULL == execute_data) {
+  if (((int)NRINI(max_nesting_level) > 0)
```

Comment:
Need to reverse comparison per style guide.
```suggestion
  if (0 < ((int)NRINI(max_nesting_level))
```